GUI/builder.py

```diff
@@ -83,6 +84,8 @@ def builder(folder, NGRAMS, GAMMA, BATCH_SIZE, LEARNING_RATE, EMBED_DIM, EPOCHS,
      	
     train_dataset, test_dataset = _setup_datasets(data=data_file, root=root, ngrams=NGRAMS, vocab=None)
 
+    print(len(train_dataset))
```

Comment:
Looks like generic debug statement. Supposed to still be here?

Reply:
I forgot about that, I'll remove it.